src/main/scala/scala/collection/decorators/IterableDecorator.scala

```diff
@@ -33,4 +33,46 @@ class IterableDecorator[C, I <: IsIterable[C]](coll: C)(implicit val it: I) {
   def lazyFoldRight[B](z: B)(op: it.A => Either[B, B => B]): B =
     it(coll).iterator.lazyFoldRight(z)(op)
 
+
+  /**
+    * Constructs an iterator where consecutive elements are accumulated as
+    * long as the output of f for each element doesn't change.
+    * <pre>
+    * Vector(1,2,2,3,3,3,2,2)
+    * .splitBy(identity)
+    * </pre>
+    * produces
+    * <pre>
+    * Iterator(Vector(1),
```

Comment:
It should be `Vector(Vector(1),` instead of `Iterator(Vector(1),`

Reply:
done